src/metric_config.hpp

```diff
@@ -24,6 +24,20 @@
 #include "status.hpp"
 
 namespace ovms {
+
+const std::string METRIC_NAME_REQUESTS_SUCCESS = "ovms_requests_success";
```

Comment:
Leave here only declaration with `extern`. Move definition into cpp file.

Reply:
Applies to all constants

Reply:
Done